app/views/projects/show.html.erb

```diff
@@ -0,0 +1,13 @@
+<h1><%= @projects.name %></h1>
+<p>Material: <%= @projects.material %></p>
+<p>Challenge Theme: <%= @projects.challenge.theme %></p>
+<p> Number of Contestants: <%= @contestants.count %></p>
```

Comment:
Careful with this - using `.count` on an association will actually change the SQL and ask the database for this information. For things like `.count`, we actually want it to appear in the Model as a method, then we can call that method here, just like you did with `average_age`.